apps/dashboard/src/app/upload/upload-files/upload-files.component.scss

```diff
@@ -29,7 +29,7 @@ h1 {
 }
 
 .dropzone {
-  margin-bottom: 30px;
+  margin-bottom: rhythm(4);
   border: 2px dashed #ccc;
   color: rgba(0, 0, 0, 0.5);
   min-height: 94px;
```

Comment:
I see you updated 30px for the rhythm equivalent, you can also update 94px for `rhythm(12)`